src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/CovariantEquals/context/defaultContext.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/Test.java b/Test.java
+index cfa1cc7..6011bfc 100644
+--- a/Test.java
++++ b/Test.java
+@@ -5,9 +5,6 @@ public class Test {
+         return false;
+     }
+ 
+-    public boolean equals(Object i) {
```

Comment:
delete equals(Object i), should have violation, but not now.

Reply:
not sure how to catch deletion of completely different method from violation AST. Probably we should implement property similar to neverSuppressedChecks https://github.com/checkstyle/patch-filters/issues/108 in SuppressionPatchXpathFilter and put CovariantEquals check there.

Reply:
@baratali yes, I agree, maybe no other better way now.